// [[Reason: TSC simplifies import type removing resolution-mode]] ////

//// [tests/cases/conformance/node/nodeModulesImportAttributesTypeModeDeclarationEmitErrors.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,20 +1,20 @@
 
 
 //// [/.src/out/index.d.ts]
 export type LocalInterface = import("pkg", { with: { "resolution-mode": "foobar" } }).RequireInterface & import("pkg", { with: { "resolution-mode": "import" } }).ImportInterface;
-export declare const a: import("pkg").RequireInterface;
+export declare const a: import("pkg", { with: { "resolution-mode": "foobar" } }).RequireInterface;
 export declare const b: import("pkg", { with: { "resolution-mode": "import" } }).ImportInterface;
 
 //// [/.src/out/other.d.ts]
 export type LocalInterface = import("pkg", { with: {} });
-export declare const a: any;
-export declare const b: any;
+export declare const a: import("pkg", { with: {} });
+export declare const b: import("pkg", { with: {} });
 
 //// [/.src/out/other2.d.ts]
 export type LocalInterface = import("pkg", { with: { "bad": "require" } }).RequireInterface & import("pkg", { with: { "bad": "import" } }).ImportInterface;
-export declare const a: import("pkg").RequireInterface;
-export declare const b: any;
+export declare const a: import("pkg", { with: { "bad": "require" } }).RequireInterface;
+export declare const b: import("pkg", { with: { "bad": "import" } }).ImportInterface;
 
 //// [/.src/out/other3.d.ts]
 export type LocalInterface = import("pkg", { with: {} })[{
     "resolution-mode": "require";
@@ -23,15 +23,15 @@
 export declare const b: invalid;
 
 //// [/.src/out/other4.d.ts]
 export type LocalInterface = import("pkg", { with: {} });
-export declare const a: any, Attribute1: invalid, RequireInterface: invalid;
-export declare const b: any, Attribute2: invalid, ImportInterface: invalid;
+export declare const a: import("pkg", { with: {} }), Attribute1: invalid, RequireInterface: invalid;
+export declare const b: import("pkg", { with: {} }), Attribute2: invalid, ImportInterface: invalid;
 
 //// [/.src/out/other5.d.ts]
 export type LocalInterface = import("pkg", { with: {} }).RequireInterface & import("pkg", { with: {} }).ImportInterface;
-export declare const a: import("pkg").RequireInterface;
-export declare const b: any;
+export declare const a: import("pkg", { with: {} }).RequireInterface;
+export declare const b: import("pkg", { with: {} }).ImportInterface;
 
 /// [Errors] ////
 
 error TS2468: Cannot find global value 'Promise'.
